pkgs/applications/window-managers/openbox/default.nix

```diff
@@ -22,15 +27,37 @@ stdenv.mkDerivation rec {
     pango imlib2
   ];
 
-  pythonPath = with python2.pkgs; [
+  pythonPath = with python3.pkgs; [
     pyxdg
   ];
 
-  src = fetchurl {
-    url = "http://openbox.org/dist/openbox/${pname}-${version}.tar.gz";
-    sha256 = "1xvyvqxlhy08n61rjkckmrzah2si1i7nmc7s8h07riqq01vc0jlb";
+  src = fetchFromGitHub {
+    owner = "Mikachu";
+    repo = "openbox";
+    rev = "release-${version}";
+    sha256 = "0k9ad731xf3l1zi3mf6ac80pal9azz4siwsv1zzg9qniksm2v0fn";
   };
 
+  patches = [
+    # Add python3 support
+    (fetchpatch {
```

Comment:
Ok, I think renames work with `fetchurl`, let's switch to that as an exception.

Reply:
Ah, this helps avoid a lot of the problems. Seems everything falls into place now.